app/src/main/res/layout/chatbox.xml

```diff
@@ -76,25 +77,24 @@
             android:contentDescription="@string/send_content_desp"
             android:padding="4dp"
             android:src="@drawable/ic_send_24dp"
-            android:tint="#757575" />
+            android:tint="@color/colorAccent" />
 
     </LinearLayout>
 
     <LinearLayout
         android:id="@+id/composeStatus"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
         android:layout_gravity="center"
         android:background="@color/colorPrimary"
         android:gravity="center|center_vertical"
+        android:orientation="horizontal"
```

Comment:
This happens on **formatting** the code. Android studio sorts the attributes in a particular order.